test.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable */
```

Comment:
I assume this is just a dummy test for local testing? Don't forget to remove it ;)

Reply:
yes, I removed it.